base/card-api.gts

```diff
@@ -915,7 +925,7 @@ function makeDescriptor<CardT extends CardConstructor, FieldT extends CardConstr
       for (let computedFieldName of Object.keys(getComputedFields(this))) {
         deserialized.delete(computedFieldName);
       }
-      (async () => await recompute(this))();
+      this[cardSettled] = this[cardSettled].then(() => recompute(this));
```

Comment:
I'd like to get some feedback on this. This is related to the issue around promises for our recomputes just floating in space unwatched. In order to be able to deal with intentional `NotLoaded` errors coming out of the field loading process I had to deal with this. I added a class property (using a symbol) for each card instance that chains the recompute promises together. I ran into issues trying to drop a recompute promises from the outside of the `recompute()` function where a promise was not the "current" one so i chained them here instead (we still drop promises within the `recompute()` function though).

Reply:
instead of chaining them, push them into a list and then wait for them all to be done (so that it is not in series)